mb-solr/pom.xml

```diff
@@ -56,22 +56,6 @@
 			<version>9.0.0</version>
 			<scope>provided</scope>
 		</dependency>
-		<dependency>
-			<groupId>jakarta.xml.bind</groupId>
-			<artifactId>jakarta.xml.bind-api</artifactId>
-			<version>3.0.1</version>
-		</dependency>
-		<!-- https://mvnrepository.com/artifact/javax.activation/activation -->
-		<dependency>
-			<groupId>javax.activation</groupId>
-			<artifactId>activation</artifactId>
-			<version>1.1.1</version>
-		</dependency>
-		<dependency>
-			<groupId>org.glassfish.jaxb</groupId>
-			<artifactId>jaxb-runtime</artifactId>
-			<version>3.0.2</version>
-		</dependency>
```

Comment:
Because these are runtime dependencies, I think I was able to successfully compile without these dependencies. I removed them to reduce confusion based on compilation and execution. 
TODO: still need to check if we need a `test` scope to be able to run the mbsolr tests